core/templates/dev/head/pages/exploration-editor-page/editor-navigation/editor-navigation-dropdown.directive.html

```diff
@@ -0,0 +1,151 @@
+<div class="dropdown hidden-md hidden-lg ng-cloak" style="position: absolute; right: 100px; top:10px;">
```

Comment:
Here and elsewhere: please use CSS rules for styling, don't use inline styles. See other files for naming conventions and make sure that any styles you introduce have an "oppia-" prefix. Avoid use of !important -- see https://github.com/oppia/oppia/wiki/Coding-style-guide#css. Also, please ensure consistent formatting, e.g. have a single space after each colon.

All these comments apply throughout this PR.

Reply:
I have removed the !important and removed most of the inline styling which I added.